hypertrace-core-graphql-gateway-service-utils/src/main/java/org/hypertrace/core/graphql/utils/gateway/UnwrappedValueConverter.java

```diff
@@ -21,6 +24,17 @@
       case DOUBLE:
         return Single.just(value.getDouble());
       case TIMESTAMP:
+        if (!StringUtils.isEmpty(value.getTimestampUnit())) {
```

Comment:
```suggestion
        if (!value.getTimestampUnit().isEmpty()) {
```
boom! down with lang (proto getters will never return null)